src/server/KbnServer.js

```diff
@@ -76,6 +77,11 @@ module.exports = class KbnServer {
     await fromNode(cb => server.start(cb));
     await require('./pid')(this, server, config);
 
+    if (isWorker) {
+      // help parent process know when we are ready
+      process.send(['WORKER_LISTENING']);
```

Comment:
The aforementioned constants should be publicly accessible so one can be used here as well.